helm-charts/sep-service/example_values.yaml

```
@@ -120,8 +120,9 @@ stellar:
         signingSeed: ${SEP10_SIGNING_SEED}
         requireKnownOmnibusAccount: false
         omnibusAccountList:
-          # - GDOHXZYP5ABGCTKAEROOJFN6X5GY7VQNXFNK2SHSAD32GSVMUJBPG75E
-          # - GC5KRPVW4TFA6ORIGTOFM3DEG6DSLACRXGDN3LNCAI7IPAGIWMVXUHVS 
+        #omnibusAccountList: "\
+        #    GAB24L6CPBKSMQOBXJ5PRA6I66X4TGZHPAEI2CMPNLC6I2WX646OYCAV,\
+        #    GAB24L6CPBKSMQOBXJ5PRA6I66X4TGZHPAEI2CMPNLC6I2WX646OYCAA"
```

Can I suggest that we use the following format?

```suggestion
       omnibusAccountList: |
            GAB24L6CPBKSMQOBXJ5PRA6I66X4TGZHPAEI2CMPNLC6I2WX646OYCAV
            GAB24L6CPBKSMQOBXJ5PRA6I66X4TGZHPAEI2CMPNLC6I2WX646OYCAA
            
```

---

good suggestion, that looks a lot cleaner. added this but with an additional comma in c6ef400